odoo-shippable/scripts/build-image.sh

```diff
@@ -186,8 +186,8 @@ EOF
 
 # Install and configure YouCompleteMe
 VIM_YOUCOMPLETEME_PATH="${HOME}/.vim/bundle/YouCompleteMe"
-git_clone_copy "${VIM_YOUCOMPLETEME_REPO}" "master" "." "${VIM_YOUCOMPLETEME_PATH}"
-(cd "${VIM_YOUCOMPLETEME_PATH}" && ./install.py)
+git clone ${VIM_YOUCOMPLETEME_REPO} ${VIM_YOUCOMPLETEME_PATH}
+(cd "${VIM_YOUCOMPLETEME_PATH}" && git reset --hard 89beeb518ef0a85346b6c65e13baab24dcaef37c && git submodule update --init --recursive && ./install.py)
```

Comment:
Add a comment to know why we are adding here...

Maybe:

 - Force to use a old version because newest version (since Valloric/YouCompleteMe#2676) requires GCC update that we don't want it for now